server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -293,6 +301,26 @@ public List<FlowInstanceDetailResp> createIndividualFlowInstance(CreateFlowInsta
     @EnablePreprocess
     @Transactional(rollbackFor = Throwable.class, propagation = Propagation.REQUIRED)
     public List<FlowInstanceDetailResp> create(@NotNull @Valid CreateFlowInstanceReq createReq) {
+        if (createReq.getTaskType() == TaskType.APPLY_DATABASE_PERMISSION) {
```

Comment:
why shouldn't we use `Preprocessor` to do this?

Reply:
`Preprocessor` now is used to check and auto fill-in task parameter. It can not split a create flow instance request to multiple. By the way, it is not necessary.

Reply:
does this cost too much time if i get too many databases